app/views/idv/by_mail/letter_enqueued/show.html.erb

```diff
@@ -19,21 +19,33 @@
 
 <%= render PageHeadingComponent.new.with_content(t('idv.titles.come_back_later')) %>
 
-<p><%= t('idv.gpo.will_send_to') %></p>
+<p><%= t('idv.gpo.will_send_to', app_name: APP_NAME) %></p>
 
 <div class="padding-y-1">
   <% @presenter.address_lines.each do |address_line| %>
     <p class="margin-bottom-0"><%= address_line %></p>
   <% end %>
 </div>
 
-<p class="padding-top-3 padding-bottom-3">
-  <%= t('idv.messages.come_back_later_html') %>
-</p>
+<% if @presenter.has_sp? %>
```

Comment:
I was working on a companion to this story with @lmgeorge. We came up with a `@presenter.show_contact_sp_instructions?` method which made things a little more self-documenting.

We also did not add a list to the YAML file. We had a descriptor for each one.

When we see that PR up we should work on squaring those up since they are essentially the same thing.

Reply:
Thanks! I've matched this naming.

I had initially assumed these were using the same presenter, but they are not. I think I'm OK duplicating the pattern / 1-liner method rather than trying to extract stuff like this:

```ruby
  def sp_name
    sp&.friendly_name
  end
```